helpers/sessions.go

```diff
@@ -0,0 +1,24 @@
+package helper
+
+import (
+	"net/http"
+
+	"github.com/gin-contrib/sessions"
+	"github.com/gin-gonic/gin"
+)
+
+func SetSession(c *gin.Context, userID uint) error {
+	session := sessions.Default(c)
+	session.Set("userID", userID)
+	if err := session.Save(); err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": "Failed to save session"})
+		return err
+	}
+	return nil
```

Comment:
For me, the spacing would be great here
```suggestion
	}

	return nil
```